### PR TITLE
Resource xrootd is added to announce downtimes

### PR DESCRIPTION
**Pull request by bockjoo**
There is no way to add xrootd downtime. So I am registering the xrootd to the xml

### CODE REVIEW

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 1]
```diff
@@ -218,6 +218,49 @@ Resources:
...
+          ID: 
+          Name: University of Florida CMS Tier 2 Admins
+      Security Contact:
+        Primary:
+          ID: 
+          Name: Bockjoo Kim
+    Description: Xrootd Redirector
+    Disable: false
+    FQDN: cmsio5.rc.ufl.edu
+    ID:
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]:
```suggestion
    ID: 1000
```

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 2]
```diff
@@ -218,6 +218,49 @@ Resources:
...
+        Primary:
+          ID: 
+          Name: Bockjoo Kim
+      Miscellaneous Contact:
+        Primary:
+          ID: 
+          Name: University of Florida CMS Tier 2 Admins
+      Security Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]:
```suggestion
          ID: 2d9012f731159dd45a082164c6aac6577167e725
```

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 3]
```diff
@@ -218,6 +218,49 @@ Resources:
...
+        Primary:
+          ID: 
+          Name: Bockjoo Kim
+      Resource Report Contact:
+        Primary:
+          ID: 
+          Name: Bockjoo Kim
+      Miscellaneous Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]:
```suggestion
          ID: 8fe996dae2ee3abd17af9f8c0f57fa6d166be5ff
```

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 4]
```diff
@@ -218,6 +218,49 @@ Resources:
...
+  UFlorida-XRD:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 
+          Name: Bockjoo Kim
+      Resource Report Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]:
```suggestion
          ID: 2d9012f731159dd45a082164c6aac6577167e725
```

File: topology/University of Florida/UF HPC/UFlorida-HPC.yaml [hunk 5]
```diff
@@ -218,6 +218,49 @@ Resources:
       StorageCapacityMax: 616
       StorageCapacityMin: 0
       TapeCapacity: 0
+  UFlorida-XRD:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID:
```
Comment (brianhlin) [inline topology/University of Florida/UF HPC/UFlorida-HPC.yaml]:
```suggestion
          ID: 2d9012f731159dd45a082164c6aac6577167e725
```

**Comment by bockjoo** [comment]
Thanks! It appeared in the downtime generator.

**Pull request merged**